### ISSUE TITLE
Update link in Github config

### ISSUE THREAD

**Issue by madiedgar** [legesher/language-legesher-python#20]
**Update Github Config Link**
The link in the `.github/config.yml` is incomplete. Please update the link to include the `.html` extension at the end of the url. 

It should look like this:

``` yml
We'd love to celebrate you and your contribution to the community. Let us say "Thank You" again, by sending you a [special gift](https://legesher.io/dreamteam.html) only given to special individuals like you who have chosen to give back.
```

**Comment by rajitha1998** [comment]
Hi @madiedgar,  I would like to work on this, can you assign me :)

**Comment by madiedgar** [comment]
Absolutely @rajitha1998 ! Thank you so much!

**Comment by rajitha1998** [comment]
I went through the repository but couldn't find a  `.github/config.yml` to update. It would be nice if you could help me with this :). Hope this screen shot helps.  @madiedgar 

![Screenshot from 2019-12-06 22-55-01](https://user-images.githubusercontent.com/36359382/70342894-f2b1bd00-187b-11ea-8cc3-3b120c99d1db.png)

**Comment by madiedgar** [comment]
Hey @rajitha1998! Thanks for pointing that out and you're completely right. 👍 Let's make this update to the `update-config` branch, as that branch has the config file needed. Also , don't worry if the PR fails in the CI checks, we're still waiting on a response from the Atom team to help with this repo. 

Thank you so much for looking into this!

**Comment by madiedgar** [comment]
Closing this as @rajitha1998 fixed this in #21 🎉
